controller/static/bower.json

```diff
@@ -21,6 +21,7 @@
     "angular-resource": "~1.3.15",
     "angular-jwt": "~0.0.6",
     "jquery-tablesort": "*",
-    "angular-sanitize": "~1.3.15"
+    "angular-sanitize": "~1.3.15",
+    "angular-semantic-ui": "~0.0.2"
```

Comment:
We're only using a simple dropdown directive from it -- we'd probably end up implementing a dropdown directive ourselves anyway eventually to clean up the dropdown usage (similar to what we had in V2).